include/knowhere/comp/knowhere_config.h

```diff
@@ -40,6 +40,17 @@ class KnowhereConfig {
     static std::string
     SetSimdType(const SimdType simd_type);
 
+    /**
+     *The purpose of this interface is: part of the sealed indexes default to using bf16 as the base data to achieve
+     *higher capacity; to ensure consistency in computation between growing and sealed, it is necessary to maintain the
+     *same precision in growing calculations as in sealed.
+     */
+    static void
+    EnablePatchForComputeFP32AsBF16();
```

Comment:
Another interesting thing to mention is that will this change the behavior of all segments on a QN in Milvus, especially in the multi-tenant situation?

Reply:
also the simhook approach